di-bridge/src/main/java/br/com/moov/dibridge/MovieDetailComponent.kt

```diff
@@ -19,23 +19,12 @@ import br.com.moov.bookmark.movie.BookmarkMovieUseCase
 import br.com.moov.bookmark.movie.UnBookmarkMovieUseCase
 import br.com.moov.core.ImageUrlResolver
 import br.com.moov.moviedetails.navigation.MovieDetailsNavigator
-import br.com.moov.movies.navigation.MoviesNavigator
 import retrofit2.Retrofit
 
-interface CoreComponent : FeatureApisProvider, CoreApisProvider
-
-interface CoreApisProvider {
+interface MovieDetailsDependencies {
```

Comment:
File should be `MovieDetailsDependencies`, right?

Reply:
right, let me fix that